modules/seqexec/server/src/main/scala/seqexec/server/keywords/package.scala

```diff
@@ -260,15 +252,13 @@ package object keywords {
   // Keywords are read and they can fail or be missing
   // This Operation will preserve the value if defined or use the default
   // In case it either fails or is empty
-  implicit class FunctorSafeOps[F[_]: ApplicativeError[?[_], Throwable], A](v: F[Option[A]]) {
-    def safeVal: F[Option[A]] = v.attempt.map {
+  implicit class KeywordValueSafeOps[F[_]: ApplicativeError[?[_], Throwable], A: DefaultHeaderValue](v: F[Option[A]]) {
```

Comment:
`DefaultHeaderValue` is not needed here.

Reply:
I merged `FunctorSafeOps` and `FunctorSafeDefaultOps` into one implicit class as `safeVal` can be made private. `safeValOrDefault` needs `A: DefaultHeaderValue`